Hackney.Shared.HousingSearch/Domain/Asset/Asset.cs

```diff
@@ -11,24 +11,36 @@ public class Asset
         }
 
         public static Asset Create(string id, string assetId, string assetType,
-            bool isAssetCautionaryAlerted, AssetAddress assetAddress, Tenure tenure)
+            bool isAssetCautionaryAlerted, AssetAddress assetAddress, Tenure tenure,
+            AssetCharacteristics assetCharacteristics, string AssetStatus, int numberOfBedSpaces, int numberOfCots,
+            bool groundFloor, bool privateBathroom, bool privateKitchen, bool stepFree)
         {
-            return new Asset(id, assetId, assetType, isAssetCautionaryAlerted, assetAddress, tenure);
+            return new Asset(id, assetId, assetType, isAssetCautionaryAlerted, assetAddress, tenure, assetCharacteristics, AssetStatus, 
+                numberOfBedSpaces, numberOfCots, groundFloor, privateBathroom, privateKitchen, stepFree);
         }
 
         public Asset()
         {
 
         }
         private Asset(string id, string assetId, string assetType,
-            bool isAssetCautionaryAlerted, AssetAddress assetAddress, Tenure tenure)
+            bool isAssetCautionaryAlerted, AssetAddress assetAddress, Tenure tenure, AssetCharacteristics assetCharacteristics, string assetStatus, 
+            int numberOfBedSpaces, int numberOfCots, bool groundFloor, bool privateBathroom, bool privateKitchen, bool stepFree)
         {
             Id = id;
             AssetId = assetId;
             AssetType = assetType;
             IsAssetCautionaryAlerted = isAssetCautionaryAlerted;
             AssetAddress = assetAddress;
             Tenure = tenure;
+            AssetCharacteristics = assetCharacteristics;
+            AssetStatus = assetStatus;
+            NumberOfBedSpaces = numberOfBedSpaces;
```

Comment:
Shouldn't some of those properties (e.g. number of bed spaces, cots, etc) be part of the assetCharacteristics object?

Reply:
Yes we are aware of those. It's just that we already had the new fields merged to main from this PR https://github.com/LBHackney-IT/housing-search-shared/pull/28 so we thought we'd continue with this to get our frontend unblocked and have a ticket on the board to do this refactoring next https://hackney.atlassian.net/browse/HT20-248